### PR TITLE
adding image label in 

### PR DESCRIPTION
**Pull request by NaomiLea**

### CODE REVIEW

File: frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-left.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const DEFAULT_PROPS:Props = {
+}
+
+const STR_REPLACE ="Replace";
+const STR_DELETE = "Delete";
+const STR_PREMIUM ="Premium Image";
+const STR_IMAGENAME = "Image name";
+const STR_DESCRIPTION = "Image description"
+
+export const LeftLabel = (props?:Props) => {
```
Comment (dakom) [inline frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-left.ts:27]: props should be `Partial<Props>` - see https://docs.sandbox.jicloud.org/book/frontend/elements-and-components/guideline-components.html#provide-arguments

File: frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import {Age} from "~/components/entry/imagelabel/admin/images/lists/age";
+import {Stream} from "~/components/entry/imagelabel/admin/images/lists/stream";
+import "@elements/core/inputs/text-underline";
+import "@elements/core/lists/list-two-column";
+import "@elements/core/images/ji";
+export default {
+  title: 'Admin/Images/Settings',
+}
+
+const STR_USED = "To be used only for";
```
Comment (dakom) [inline frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts:11]: Does this ever change at runtime? If not - should be in an element

File: frontend/storybook/src/components/entry/imagelabel/one.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+interface Args {
+}
+
+const DEFAULT_ARGS:Args = {
+}
+
+
+const STR_NEXT = "Next";
```
Comment (dakom) [inline frontend/storybook/src/components/entry/imagelabel/one.ts:18]: Do these ever change at runtime? If not - should be in an element

File: frontend/storybook/src/components/entry/imagelabel/one.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+const STR_LABEL ="Label Images"
+
+export const ImageLabelFullOne = (props?:Args) => {
```
Comment (dakom) [inline frontend/storybook/src/components/entry/imagelabel/one.ts:21]: Args should be `Partial<Args>` see:
https://docs.sandbox.jicloud.org/book/frontend/elements-and-components/guideline-components.html#provide-arguments

**Review by dakom — CHANGES_REQUESTED** [review]

File: frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts (hunk 2 above)
Comment (NaomiLea) [reply frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts:11]: Being used in an existing dynamic element and this component changes in runtime

File: frontend/storybook/src/components/entry/imagelabel/one.ts (hunk 3 above)
Comment (NaomiLea) [reply frontend/storybook/src/components/entry/imagelabel/one.ts:18]: This is the text for the button that does change in runtime I believe. Added from existing element.

Comment (dakom) [reply frontend/storybook/src/components/entry/imagelabel/one.ts:18]: OK - fine for now, will be refactored later with https://github.com/ji-devs/ji-cloud/issues/769

File: frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-left.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import "@elements/entry/admin/images/meta/sections/left";
```
Comment (dakom) [inline frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-left.ts:1]: The file location here should match the storybook - `entry/admin/images/meta/...`

File: frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import {ImageStyle} from "~/components/entry/imagelabel/admin/images/lists/image-style";
```
Comment (dakom) [inline frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts:1]: The file location here should match the storybook - `entry/admin/images/meta/...`

File: frontend/storybook/src/components/entry/imagelabel/one.ts [hunk 7]
```diff
@@ -0,0 +1,38 @@
+import "@elements/entry/admin/images/meta/page";
```
Comment (dakom) [inline frontend/storybook/src/components/entry/imagelabel/one.ts:1]: The file location here should match the storybook - `entry/admin/images/meta/...`

**Review by dakom — CHANGES_REQUESTED** [review]

**Pull request merged**